xamarin/realm-tutorial-dotnet/App.xaml.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Xamarin.Forms;
+using System.IO;
+using System.Xml;
+
+namespace RealmTemplateApp
```

Comment:
1. Changing namespaces in VS is notoriously a PITA.
2. You can't omit namespaces; they are a fundamental part of class naming.
3. We already have `App` as a default class and `App` as a default Realm class. Naming the namespace App will open a hole in the space-time continuum.

Reply:
Maybe we can call it something else? `Main`, `Client`, I don't know. Just a suggestion.

Reply:
The namespace defaults to the project name itself, and changing it would probably be odd to .NET devs. Unless you feel really strongly, I'd like to leave it as-is.

Reply:
Can we change the project name, then? "TodoApp" perhaps?